docs/index.md

```diff
@@ -3,21 +3,13 @@
 
 ## Introduction
 
-- This is a collection of info pages about 'doing stuff' using the command line. 
+- This is a collection of info pages about the command line and tools that can be used with it. 
 
-## Get started with the CLI
+## Content
 
-- [First Steps with the CLI](cli)
-
-
-## Get started with Python 
-
-- [First steps in Python](python)
-
-
-## Get started with Git 
-
-- [First steps with Git](git)
+- [Getting started with the command line](cli)
```

Comment:
Prima! 
(Overigens kan ik me voorstellen dat we de studenten bij de eerste intro de link rechtstreek naar de betreffende page geven in plaats van starten vanaf deze start page).